README.md

```diff
@@ -1,16 +1,31 @@
 # national-flow-observations
-This repository pulls national flow data from NWIS. The pipeline pushes to a shared cache in an S3 bucket called [ds-pipeline-national-flow-observations](https://s3.console.aws.amazon.com/s3/buckets/ds-pipeline-national-flow-observations/?region=us-west-2&tab=overview).
+This repository pulls national flow data from NWIS for 50 states and all territories. The pipeline pushes to a shared cache in an S3 bucket called [ds-pipeline-national-flow-observations](https://s3.console.aws.amazon.com/s3/buckets/ds-pipeline-national-flow-observations/?region=us-west-2&tab=overview).
```

Comment:
You'll have to be logged into the console already for this link to work.  If you aren't it directs you to the non-gov AWS login page which I'm not sure works for us.